index.html

```diff
@@ -15,14 +15,22 @@
             <nav class="logo-show">
               <h2>My Logo</h2>
             </nav>
-            <nav class="menu">
-              <nav><img src="photo/navs/Union.png" alt=""></nav>
-            </nav>
+            <div class="menu">
+             <button class="btn-menu"><img src="photo/navs/Union.png" alt=""></button> 
+            </div>
+            <div class="t-menu">
+              <button class="c-button">&times;</button>
+              <ul class="tul">
+                <li><a href="tul-1">Portfolio</a></li>
+                <li><a href="tul-1">About</a></li>
+                <li><a href="tul-1">Contact</a></li>
```

Comment:
[ Required ] Great job here but it still doesn't work, As I said here I think you shoud reference to each section with `id` so the first thing you should put in each section an `id` then 

[ Hint] you can put `href` to each `id` like this

![image](https://user-images.githubusercontent.com/58553711/180915141-8c41c882-7ccd-44ad-a0c4-d2fd69d09eb0.png)


- and for each section pass the id
![image](https://user-images.githubusercontent.com/58553711/180915337-e51405a8-1cc2-4be2-8ad0-b263a74d043d.png)

- And please after taking to the specific section the menu should be closed so make sure to close it